contracts/SafientMain.sol

```diff
@@ -109,7 +116,7 @@ contract SafientMain is Safes, Claims, Guardians, IArbitrable {
         Types.RecoveryProof[] memory _guardianproof,
         string[] memory _secrets,
         string memory _safeId
-    ) internal returns (bool) {
```

Comment:
Does this have to be `external `?

Reply:
Yeah, else the function is not available for making contract calls